### DISCUSSION TITLE
Camel-jbang 4.7.0 has no --logging-config-path option

### DISCUSSION THREAD

**Discussion by javatechno** [apache/camel-karavan#1394] [Q&A] [answered]
### Ask a question

How can I specify path to log4j2.properties file in the latest camel-jbang version?
AFAIK, the latest version, which supported CMD option --logging-config-path was 4.6.0, now it is missing. What option should I pass to camel-jbang cmd to is fetches my custom log42 configuration?

### Variant

Web Application

### Container Management (if applicable)

Docker

### Operating System (if applicable)

Linux

### Version

4.7.0

**Comment by javatechno** [comment]
Same for --logging-level flag

**Comment by davsclaus** [comment, marked as answer]
Use `camel run --help` to see the options and these options are included.
